src/SqlAsyncEnumerable.cs

```diff
@@ -112,7 +112,7 @@ private async Task<bool> GetNextRowAsync()
                             this._reader = await command.ExecuteReaderAsync();
```

Comment:
Since ExecuteReaderAsync has issues with cancellation - I would recommend changing that to ExecuteReader too. Do you plan to do that separately?

Reply:
That goes for any Async calls from SqlClient, right? @MaddyDev Make sure you're doing a search all for anything with "Async" in its name so we aren't missing any others.